### PR TITLE
Update workers/ from monorail results

### PR DESCRIPTION
**Pull request by stephenmcgruer**
See https://github.com/web-platform-tests/wpt-metadata/issues/798 for
the methodology

### CODE REVIEW

File: workers/META.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1071345
+      results:
+        - test: dedicated-worker-parse-error-failure.html
+    - product: chrome
+      url: https://crbug.com/906850
```
Comment (foolip) [inline workers/META.yml:7]: This bug doesn't link to opaque-origin.html but rather a sandboxed-iframe-with-opaque-origin.html

File: workers/META.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      results:
+        - test: opaque-origin.html
+    - product: chrome
+      url: https://crbug.com/795636
```
Comment (foolip) [inline workers/META.yml:11]: This is a bug about upstreaming tests, and doesn't say anything about the failure.

File: workers/constructors/SharedWorker/META.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+links:
+    - product: chrome
+      url: https://crbug.com/655458
```
Comment (foolip) [inline workers/constructors/SharedWorker/META.yml:3]: This is an umbrella bug for WPT failures, which I don't think is useful to link.

File: workers/interfaces/SharedWorkerGlobalScope/META.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/655458
```
Comment (foolip) [inline workers/interfaces/SharedWorkerGlobalScope/META.yml:3]: Umbrella bug

File: workers/modules/META.yml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/841831
```
Comment (foolip) [inline workers/modules/META.yml:3]: Looks correct!

File: workers/META.yml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1071345
```
Comment (foolip) [inline workers/META.yml:3]: Looks right!

**Review by foolip — COMMENTED** [review]
These initial results aren't super promising. Maybe it would make sense to get the titles of all bugs and eliminate ones which are clearly umbrella bugs?

File: workers/META.yml (hunk 1 above)
Comment (stephenmcgruer) [reply workers/META.yml:7]: Interesting, it does get returned from monorail search (https://bugs.chromium.org/p/chromium/issues/list?q=%2Fworkers%2Fopaque-origin.html&can=2)... I wonder if monorail splits on `/` and I should be quoting these search terms -_-.

Comment (stephenmcgruer) [reply workers/META.yml]: Reverted

File: workers/META.yml (hunk 2 above)
Comment (stephenmcgruer) [reply workers/META.yml]: Reverted

File: workers/constructors/SharedWorker/META.yml (hunk 3 above)
Comment (stephenmcgruer) [reply workers/constructors/SharedWorker/META.yml]: Reverted

File: workers/interfaces/SharedWorkerGlobalScope/META.yml (hunk 4 above)
Comment (stephenmcgruer) [reply workers/interfaces/SharedWorkerGlobalScope/META.yml]: Reverted

**Pull request merged**